variables.tf

```diff
@@ -54,6 +57,13 @@ variable "container_registry_config" {
   })
 }
 
+#azure_service_bypass
+variable "azure_services_bypass" {
+  description = "Whether to allow trusted Azure services to access a network restricted Container Registry? Possible values are None and AzureServices. Defaults to AzureServices"
```

Comment:
follow a standard, description should be at last, first `type` than `default` than `descrption`

Reply:
@pranaydeokar Please fix it.